webrtc.html

```diff
@@ -336,25 +336,25 @@
           class="simple">
             <tbody>
               <tr>
-                <th colspan="2">Enumeration description</th>
+                <th colspan="2">Enumeration description (non-normative)</th>
```

Comment:
Suggest replacing this with "Enumeration description, summarized from the normative description in (JSEP-reference)".